tests/ui/check-cfg/well-known-values.rs

```diff
@@ -106,4 +106,7 @@ fn unix() {}
 #[cfg(doc)]
 fn doc() {}
 
+#[cfg(clippy)]
+fn clippy() {}
```

Comment:
Could you also add the cfg above as to check that we only expect `clippy` and nothing else.

```diff
 #[cfg(any(
     // tidy-alphabetical-start
+    clippy = "_UNEXPECTED_VALUE",
+    //~^ WARN unexpected `cfg` condition value
     debug_assertions = "_UNEXPECTED_VALUE",
     //~^ WARN unexpected `cfg` condition value
```

Reply:
Ah sure.